packages/fsapp/src/beta-app/router/utils.ts

```diff
@@ -24,7 +24,9 @@ export const resolveRoutes = async ({
   const getChildPath = (route: Route | RouteCollection) => (
     'initialPath' in route ? route.initialPath : route.path
   );
-
+  const getPrefixedPath = (route: Route | RouteCollection, prefixedPath: string) => (
+    'initialPath' in route ? '' : prefixedPath
+  );
```

Comment:
We should probably rename these functions.
`get` implies some type of network or storage transaction which isn't hat is happening here.

The `getPrefixedPath()` is only used in one place so we could just inline that or simply put it in a variable.
I am open to suggestions but `getChildPath()` and `getPrefixedPath()` need better names.